src/index/btree_index.cpp

```diff
@@ -78,7 +98,7 @@ bool BTREE_TEMPLATE_TYPE::InsertEntry(const storage::Tuple *key,
 
 BTREE_TEMPLATE_ARGUMENT
 bool BTREE_TEMPLATE_TYPE::DeleteEntry(const storage::Tuple *key,
-                                                 const ItemPointer &location) {
+                                      const ItemPointer &location) {
```

Comment:
The major reason is that the location is only used for finding the correct index entry that points to the location referred to by `location`.